tests/kernel/run_ltp.pm

```diff
@@ -269,20 +269,28 @@ sub run {
         script_run('ps axf');
         script_run('netstat -ap');
 
+        script_run('cat /etc/resolv.conf');
+        script_run('cat /etc/nsswitch.conf');
+        script_run('cat /etc/hosts');
+
         # emulate /opt/ltp/testscripts/network.sh
         assert_script_run('TST_TOTAL=1 TCID="network_settings"; . test_net.sh; export TCID= TST_LIB_LOADED=');
 
         script_run('env');
+
+        script_run('ip addr');
+        script_run('ip route');
+        script_run('ip link');
```

Comment:
@richiejp you're right, `ip link` is redundant, going to remove it. Thanks for notice.